source/grid/grid_tools.cc

```diff
@@ -46,8 +46,11 @@
 #include <deal.II/hp/mapping_collection.h>
 #include <deal.II/numerics/matrix_tools.h>
 
+
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
 #include <boost/random/uniform_real_distribution.hpp>
 #include <boost/random/mersenne_twister.hpp>
+DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
```

Comment:
No, it's ok. I would have put that into a separate commit/patch, but it's not important.